KamerConnect.View.MAUI/Views/MatchRequestsView.xaml.cs

```diff
@@ -33,54 +74,121 @@ public void GetMatchRequests()
                 HorizontalOptions = LayoutOptions.Center,
                 Content = new Image
                 {
-                    Source = "logo.png",
+                    Source = person.ProfilePicturePath != null
+                        ? _fileService.GetFilePath(_bucketName, person.ProfilePicturePath)
+                        : "geenProfiel.png",
                     Aspect = Aspect.AspectFit, 
                     VerticalOptions = LayoutOptions.Center,
                     HorizontalOptions = LayoutOptions.Center
                 }
             };
-            var label1 = new Label { Text = $"Label {i} 2", HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center};
-            var label2 = new Label { Text = $"Label {i} 3", HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center};
-            var label3 = new Label { Text = $"Label {i} 4", HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center};
+            var label1 = new Label { Text = person.FirstName, HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center};
```

Comment:
Label1, Label2 enzov. is een erg slechte naamgeving. Ik vraag me ook nog steeds af waarom dit niet gewoon gebind is, maar dat is prima als dat beter werkt voor jou nu.